### PR TITLE
[Doc] Give more informations about comande line to build and deploy project

### PR DESCRIPTION
**Pull request by yjacolin**

### CODE REVIEW

File: doc/integrator/install_application.rst [hunk 1]
```diff
@@ -273,9 +279,8 @@ This previous command will do many things like:
 
   * download and install the project dependencies,
 
-  * adapt the application configuration to your environment,
+  * deploy the mapfish-print service
```
Comment (jwkaltz) [inline doc/integrator/install_application.rst]: but only if mapfish-print service is configured for building

File: doc/integrator/install_application.rst [hunk 2]
```diff
@@ -257,7 +257,13 @@ You can build and install the application with the command:
...
     ./docker-run make --makefile=<user>.mk build
 
-Create the application tables, and directly set the version (details later):
+This previous command will do many things like:
+
+  * adapt the application configuration to your environment,
+
+  * build the javascript and css resources into compressed files,
+
+Then create the application tables, and directly set the version (details later):
```
Comment (jwkaltz) [inline doc/integrator/install_application.rst]: I don't understand the "and directly set the version" part. I know this part was already there, before your PR. But maybe we can clarify it? If there are "details later", we should explicitly point to them.

Comment (yjacolin) [reply doc/integrator/install_application.rst]: ping  @sbrunner could you explain this? :) I will update it this afternoon (or later).

Comment (yjacolin) [reply doc/integrator/install_application.rst]: There is a table in <schema_name>_static named alembic which contains version of alembic.

Comment (sbrunner) [reply doc/integrator/install_application.rst]: For me this a relay old comment and can be removed ...

File: doc/integrator/install_application.rst [hunk 3]
```diff
@@ -257,7 +257,14 @@ You can build and install the application with the command:
...
     ./docker-run make --makefile=<user>.mk build
 
-Create the application tables, and directly set the version (details later):
+This previous command will do many things like:
+
+  * adapt the application configuration to your environment,
+
+  * build the javascript and css resources into compressed files.
+
+Then create the application tables, and directly set the alembic version in a `alembic_version`
```
[the remark this answers is not in the document]
Comment (sbrunner) [reply doc/integrator/install_application.rst]: Yes it can be removed :-)

File: doc/integrator/install_application.rst [hunk 4]
```diff
@@ -273,9 +279,8 @@ This previous command will do many things like:
 
   * download and install the project dependencies,
 
-  * adapt the application configuration to your environment,
+  * deploy the mapfish-print service if it is configured for building.
 
-  * build the javascript and css resources into compressed files,
 
 Your application should now be available at:
 ``http://<hostname>/``.
```
Comment (jwkaltz) [inline doc/integrator/install_application.rst]: IIUC https is default now
-> so https:://<hostname>

File: doc/integrator/install_application.rst [hunk 5]
```diff
@@ -257,7 +257,13 @@ You can build and install the application with the command:
 
     ./docker-run make --makefile=<user>.mk build
 
-Create the application tables, and directly set the version (details later):
+This previous command will do many things like:
+
+  * adapt the application configuration to your environment,
+
+  * build the javascript and css resources into compressed files.
```
Comment (sbrunner) [inline doc/integrator/install_application.rst]: JavaScript and CSS

File: doc/integrator/install_application.rst [hunk 6]
```diff
@@ -273,9 +279,8 @@ This previous command will do many things like:
 
   * download and install the project dependencies,
 
-  * adapt the application configuration to your environment,
+  * deploy the mapfish-print service if it is configured for building.
```
Comment (sbrunner) [inline doc/integrator/install_application.rst]: MapFish Print

**Pull request merged**